packages/sync/src/offline/retry/RetriableOperation.ts

```diff
@@ -49,7 +47,7 @@ export class RetriableOperation extends OperationQueueEntry {
             setTimeout(resolve, this.timeout);
           });
 
-          this.timeout *= 2;
+          this.timeout *= 3;
```

Comment:
minor comment: maybe this should be a constant variable at the top of the file like `TIMEOUT_MULTIPLIER`